compiler/src/dmd/backend/gflow.d

```diff
@@ -32,7 +32,13 @@ import dmd.backend.dvec;
 nothrow:
 @safe:
 
-void vec_setclear(size_t b, vec_t vs, vec_t vc) { vec_setbit(b, vs); vec_clearbit(b, vc); }
+void vec_setclear(size_t b, vec_t vs, vec_t vc)
+{
+    if (vec_testbit(b, vs) && !vec_testbit(b, vc)) return; // Avoid redundant operations
```

Comment:
Setting and clearing a bit takes a trivial amount of time, I think this added check only makes things slower.